content/docs/command-reference/remote/modify.md

```diff
@@ -35,6 +35,9 @@ manual editing could be used to change the configuration.
 - `-u`, `--unset` - delete configuration value for the given config `option`.
   Don't provide a `value` when employing this flag.
 
+- `--jobs` - number of workers started. Accepts positive integers. The default
```

Comment:
To me, it looks like I added two identical explanations of identical command options. Is there a place and need to explain configuration field?

Reply:
@I159 `dvc remote` doesn't have `--jobs` flag though. It has `jobs` config option. See `verify` in "Available parameters for all remotes" below and add the same thing for it.

Reply:
Done.